src/context/ErrorContextProvider.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactNode, useState } from 'react';
+import { ErrorValues } from '../types/ErrorValues';
+
+type Props = {
+  children: ReactNode,
+};
+
+type ContextValues = {
+  hideError: boolean,
+  setHideError: (change: boolean) => void,
+  errorMessage: ErrorValues | null,
+  setErrorMessage: (error: ErrorValues | null) => void,
+};
+
+export const ErrorContext = React.createContext({} as ContextValues);
+
+export const ErrorContextProvider: React.FC<Props> = ({ children }) => {
+  const [hideError, setHideError] = useState(true);
```

Comment:
I'd name it `isErrorHidden` - [article](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig) - just FYI